src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
```

Comment:
It seems you've set the margin for the 'body' element here, but the 'html' element also has margins set to 0. This is redundant since the 'html' element does not have margins by default. You can remove the margin property from either the 'html' or 'body' selector, but typically the 'body' selector is used for this purpose.